src/flamenco/runtime/fd_executor.h

```diff
@@ -13,6 +13,11 @@
 
 FD_PROTOTYPES_BEGIN
 
+void
+fd_create_instr_context_protobuf_from_instructions( fd_exec_test_instr_context_t * instr_context,
+                                                 fd_exec_txn_ctx_t const *txn_ctx,
+                                                 fd_instr_info_t const *instr );
+
```

Comment:
@mjain-jump when and why did we remove this decl?

Reply:
could we add a doccomment?